.vscode/settings.json

```diff
@@ -100,10 +100,8 @@
     "scoped_allocator": "cpp",
     "shared_mutex": "cpp"
   },
-  "workbench.colorCustomizations": {
```

Comment:
Isn't this block from fortiss and should be kept?

Reply:
Its from one of my plugins.